File: app/src/main/java/us/fetch321/twit/sample/MainActivity.java
package us.fetch321.twit.sample;

import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.view.View;

import us.fetch321.twit.Twit;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        findViewById(R.id.button).setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                Twit.show(MainActivity.this, "nah", "https://google.com");
            }
        });
    }
}

File: app/src/main/java/us/fetch321/twit/sample/MainActivity.kt
package us.fetch321.twit.sample

import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import kotlinx.android.synthetic.main.activity_main.*
import us.fetch321.firstlight.Attributions
import us.fetch321.firstlight.FirstLight

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        val config = FirstLight.Config(this)
                .addAttribution(Attributions.AndroidX)
                .addAttribution(Attributions.Coil)
                .addAttribution(Attributions.Firebase)
                .addAttribution(Attributions.Kotlin)
                .addAttribution(Attributions.MaterialComponents)
                .addAttribution(Attributions.Timber)
                .addAttribution(Attributions.Roboto)

        FirstLight.init(config)
        toolbar.title = "First Light"

        buttonSeeFirstLight.setOnClickListener {
            FirstLight.show(this)
        }

        buttonSeeAttribution.setOnClickListener {
            FirstLight.showAttribution(this)
        }
    }
}